BUILD.md

```diff
@@ -132,3 +132,13 @@ see how to write a new Kanister Function.
 
 Don't forget to update the documentation at `docs/functions.rst` with
 configuration information and examples to show off your new function.
+
+## Build Kanister image
+
+The Kanister build image [workflow](.github/workflows/kanister-image-build.yaml)
+is used to build and push a new Kanister build image (`ghcr.io/kanisterio/build`).
+It is an on-demand workflow and needs to be run manually. This workflow expects
+`image tag` value as an input.
+
+The author updating the build image tag must raise a separate PR to update this
+value for it to be used in the build process. It should be set it [here](https://github.com/kanisterio/kanister/blob/master/Makefile#L61).
```

Comment:
@shahpratikr 
would it not be ok if someone raises just one PR with build image Dockerfile changes and updates in the Makefile for that new tag?

Reply:
yes it would be okay.